src/Main.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import Navigation from "./components/Navigation";
+import AccomodationCard from "./components/AccomodationCard";
+import AccomodationDetails from "./components/AccomodationDetails";
+import PlaceCard from './components/PlaceCard';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import CityCard from './components/CityCard';
+
+function Main() {
+    const accomodations = [
```

Comment:
Because for example I have two objects of the "places". And also that's how the lecturer in the course handled it

> Is there a reason you declared all the data as arrays? If not, extract the objects outside of the arrays